llvm/test/CodeGen/LoongArch/calllowering-ret.ll

```diff
@@ -0,0 +1,18 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
```

Comment:
It's better to put the test and future tests in a seperate sub-folder, like `GlobalISel`.